src/controllers/examples.ts

```diff
@@ -230,6 +228,7 @@ export const CredentialExchangeRecordExample = {
 export const ProofRecordExample: ProofExchangeRecordProps = {
   id: '821f9b26-ad04-4f56-89b6-e2ef9c72b36e',
   protocolVersion: 'v2',
+  role: ProofRole.Verifier,
```

Comment:
`role` is now required https://github.com/openwallet-foundation/credo-ts/blob/e58ec5bd97043d57fcc3c5a4aee926943e6c5326/packages/core/src/modules/proofs/repository/ProofExchangeRecord.ts#L15